MyBus/MyBus/Model/Road/MapBusRoad.swift

```diff
@@ -10,39 +10,40 @@ import Foundation
 import Mapbox
 
 class MapBusRoad: NSObject {
-    var roadStopsMarkerList : [MGLPointAnnotation] = [MGLPointAnnotation]()
-    var busRoutePolylineList : [MGLPolyline] = [MGLPolyline]()
+    var roadStopsMarkerList: [MGLPointAnnotation] = [MGLPointAnnotation]()
+    var busRoutePolylineList: [MGLPolyline] = [MGLPolyline]()
 
-    func addBusRoadOnMap(roadResult : RoadResult) -> MapBusRoad
+    func addBusRoadOnMap(roadResult: RoadResult) -> MapBusRoad
     {
         addRoadMarkers(roadResult)
         addBusRoutePolyline(roadResult)
-        return self;
+        return self
     }
 
 
-    func addBusRoutePolyline(roadResult : RoadResult) -> Void {
+    func addBusRoutePolyline(roadResult: RoadResult) -> Void {
         // First bus route polyline
-        let firstBusRoute : Route = (roadResult.routeList.first)!
+        let firstBusRoute: Route = (roadResult.routeList.first)!
         let firstBusLine = busPolylineBuilder(firstBusRoute)
         busRoutePolylineList.append(firstBusLine)
 
         // If road is combinated, we add second bus route polyline
         let isCombinatedRoad = roadResult.roadResultType == 1
 
-        if (isCombinatedRoad)
+        if isCombinatedRoad
         {
-            let secondBusRoute = roadResult.routeList[1];
+            let secondBusRoute = roadResult.routeList[1]
             let secondBusLine = busPolylineBuilder(secondBusRoute)
             busRoutePolylineList.append(secondBusLine)
         }
     }
 
-    func addRoadMarkers(roadResult : RoadResult) -> Void {
-        let isSingleRoad = roadResult.roadResultType == 0
+    func addRoadMarkers(roadResult: RoadResult) -> Void {
+        let busRouteType: MyBusRouteResultType = roadResult.roadResultType == 0 ? MyBusRouteResultType.Single : MyBusRouteResultType.Combined
```

Comment:
Can we make this conditional self-contained like `let busRouteType = roadResult.busRouteResultType()`?